main.js

```diff
@@ -160,4 +160,249 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// **----------  OPDRACHT 1A ----------**
+
+const typeTvAndNames = inventory.map ((stockTv)=>{
+  return stockTv.type +" "+ stockTv.name;
+})
+ console.log(typeTvAndNames);
+
+
+// **----------  OPDRACHT 1B ----------**
+
+const soldOutTvs = inventory.filter ((stockTv)=>{
+  return (stockTv.originalStock - stockTv.sold) === 0;
+})
+console.log(soldOutTvs);
+
+
+// **----------  OPDRACHT 1C ----------**
+
+const ambilightTvs = inventory.filter ((stockTv) => stockTv.options.ambiLight === true);
```

Comment:
Omdat `stockTv.options.ambiLight` op zichzelf al een boolean is hoef je deze niet te checken op `true`
```suggestion
const ambilightTvs = inventory.filter(stockTv => stockTv.options.ambiLight);
```